src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -989,19 +993,38 @@ namespace webnn_native { namespace ie {
         } else {
             attrs.shape_calculation_mode = ngraph_shape_calc_mode::Scales;
         }
-        const ngraph_node_t* axes =
-            AddConstantWithGraph<int64_t>(precision_e::I64, {4}, {0, 1, 2, 3});
-        ngraph_node_t* resampleNode;
-        if (transpose) {
-            input = TransposeInputLayout(input, true);
+        // axes: Interpolate layer only supports resize on spatial dimensions(depth, height and
+        // width)
+        // https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp#L2515
+        const ngraph_node_t* axesNode =
+            AddConstantWithGraph<int32_t>(precision_e::I32, {2}, {2, 3});
+        // Transpose Input Layout => NCHW
+        switch (inputLayout) {
+            case InputLayout::Hwnc:
+                input = TransposeInputLayout(input, TransposeType::HwncToNchw);
+                break;
+            case InputLayout::Nhwc:
+                input = TransposeInputLayout(input, TransposeType::NhwcToNchw);
+                break;
+            default:
+                break;
```

Comment:
Maybe keep original design using `if` looks better, so that it doesn't break for NCHW layout.